server/src/main/java/org/elasticsearch/action/admin/indices/close/CloseIndexRequest.java

```diff
@@ -38,6 +40,7 @@
 
     private String[] indices;
     private IndicesOptions indicesOptions = IndicesOptions.strictExpandOpen();
+    private ActiveShardCount waitForActiveShards = ActiveShardCount.DEFAULT; //NORELEASE Changes this to NONE to keep previous behavior
```

Comment:
maybe amend the comment to say "Change this to NONE on 7.x to keep previous behavior"

Reply:
Ok